pkg/ipmi/ipmi.go

```diff
@@ -77,6 +82,34 @@ type systemInterfaceAddr struct {
 	lun      byte //nolint:unused
 }
 
+type StandardSelEventRecord struct {
```

Comment:
Are there other kinds of IPMI events? If not, why not just call it `StandardEvent`?